src/app/static/src/tests/integration/baseline.itest.ts

```diff
@@ -44,7 +45,7 @@ describe("Baseline actions", () => {
 
         expect(commit.mock.calls[1][0]["type"]).toBe("ShapeUpdated");
         expect(commit.mock.calls[1][0]["payload"]["filename"])
-            .toBe("malawi.geojson");
+            .toBe("A59C122F27303D93467C0F2D09489878.geojson");
```

Comment:
Note this ticket to return original filenames rather than hashes: https://vimc.myjetbrains.com/youtrack/issue/mrc-513